staging/src/k8s.io/api/certificates/v1beta1/types.go

```diff
@@ -153,3 +169,10 @@ const (
 	UsageMicrosoftSGC      KeyUsage = "microsoft sgc"
 	UsageNetscapeSGC       KeyUsage = "netscape sgc"
 )
+
+const (
+	KubeAPIServerClientSignerName = "kubernetes.io/kube-apiserver-client"
+	KubeletClientSignerName       = "kubernetes.io/kubelet-client"
+	KubeletServingSignerName      = "kubernetes.io/kubelet-serving"
+	LegacyUnknownSignerName       = "kubernetes.io/legacy-unknown"
+)
```

Comment:
Not sure if these consts should also be defined in the internal API version too? Conversion treats the signerName field as an opaque string right now (i.e. we never have to translate one 'legacy' signerName into a different one), so I *think* not... what is the convention with string typed consts?

Reply:
> Not sure if these consts should also be defined in the internal API version too?

external only please